CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,52 @@
+# Code of Conduct
+
+## 1. Purpose
+
+A primary goal of open-sourcing KAGOME is to be inclusive to the largest number of contributors, with the most varied and diverse backgrounds possible. As such, we are committed to providing a friendly, safe and welcoming environment for all, regardless of gender, sexual orientation, ability, ethnicity, socioeconomic status, and religion (or lack thereof).
+
+## 2. Expected Behavior
+
+The following behaviors are expected and requested of all community members:
+
+- Participate in an authentic and active way. In doing so, you contribute to the health and longevity of this community.
+- Exercise consideration and respect in your speech and actions.
+- Attempt collaboration before conflict.
+- Refrain from demeaning, discriminatory, or harassing behavior and speech.
+- Be mindful of your surroundings and of your fellow participants.
+
+## 3. Unacceptable Behavior
+
+The following behaviors are considered harassment and are unacceptable within our community:
+
+- Violence, threats of violence or violent language directed against another person.
+- Sexist, racist, homophobic, transphobic, ableist or otherwise discriminatory jokes and language.
+- Posting or displaying sexually explicit or violent material.
+- Personal insults, particularly those related to gender, sexual orientation, race, religion, or disability.
+
+## 4. Consequences of Unacceptable Behavior
+
+Unacceptable behavior from any community member will not be tolerated. Anyone asked to stop unacceptable behavior is expected to comply immediately.
+
+If a community member engages in unacceptable behavior, the project maintainers may take any action they deem appropriate, up to and including a temporary ban or permanent expulsion from the community without warning.
+
+## 5. Reporting Guidelines
+
+If you are subject to or witness unacceptable behavior, or have any other concerns, please notify a community moderator as soon as possible.
+
+## 6. Addressing Grievances
+
+If you feel you have been falsely or unfairly accused of violating this Code of Conduct, you should notify the project maintainers with a concise description of your grievance.
+
+## 7. Scope
+
+This Code of Conduct applies both within project spaces and in public spaces when an individual is representing the project or its community.
+
+## 8. Contact info
+
+Quadrivium Numera Pte. Ltd. can be reached at github[at]qdrvm.io
```

Comment:
> github[at]qdrvm.io

May be it is already done. If not, then I suggest to set up a redirect for all incoming mail from this address to someone's primary email, otherwise there is a chance of missing something or not being on time.

Reply:
yes, we do redirect messages from this email to the employees email